src/portable/ReactShared/Util.cs

```diff
@@ -179,5 +180,15 @@ public static void DeleteFolder(string fullPath)
 			}
 		}
 
+		public static TimeSpan GetAudioDuration(string filePath)
+		{
+			using (var shell = Microsoft.WindowsAPICodePack.Shell.ShellObject.FromParsingName(filePath))
```

Comment:
My concern is that since this is accessing the Windows Shell, it won't work on Linux. I think we will have to test it with Linux before we should commit it.